modules/irc/templates/pywikibot/user-config.py

```diff
@@ -1,5 +1,6 @@
+# MANAGED BY PUPPET
 family = 'wikitide'
-mylang = 'meta'
-usernames['wikitide']['meta'] = 'BeeBot'
-authenticate['*.miraheze.org'] = ('<%= @consumer_token %>', '<%= @consumer_secret %>', '<%= @access_token %>', '<%= @access_secret %>')
+mylang = 'metawiki'
+usernames['wikitide']['*'] = 'BeeBot',
+authenticate['*'] = ('<%= @consumer_token %>', '<%= @consumer_secret %>', '<%= @access_token %>', '<%= @access_secret %>')
```

Comment:
does this even work? Pywikibot says "Pywikibot supports wildcard (*) in the prefix of hostname" (https://github.com/wikimedia/pywikibot/blob/2f678b61e6b8c032c3a5f3e4e6a600f6284d9ac7/pywikibot/config.py#L203C6-L203C63), nothing about just setting it to `*` even in the examples.